salt/_states/docker_registry.py

```diff
@@ -0,0 +1,185 @@
+'''
+States to manage images in a Docker registry.
+'''
+
+import logging
+
+import salt.exceptions
+
+log = logging.getLogger(__name__)
+
+
+__virtualname__ = 'docker_registry'
+
+
+def __virtual__():
+    if 'docker_registry.get_image_manifest' not in __salt__:
```

Comment:
Not an expert on Salt, so I may be wrong, but could not we just check for ` if 'docker_registry' not in __salt__:` to see if the module is available? or do we have to check for a specific method?

Reply:
It's how I saw other states do it.